src/core/data.jl

```diff
@@ -192,6 +192,10 @@ function _make_per_unit(data::Dict{String,Any}, mva_base::Real)
         apply_func(branch, "rate_b", rescale)
         apply_func(branch, "rate_c", rescale)
 
+        apply_func(branch, "c_rating_a", rescale)
+        apply_func(branch, "c_rating_b", rescale)
+        apply_func(branch, "c_rating_c", rescale)
```

Comment:
I prefer seeing current parameters being rescaled in their own base, e.g.
```
ka_base = mva_base/kv_base
rescale_ampere      = x -> x/ka_base
apply_func(branch, "c_rating_a", rescale_ampere)
apply_func(branch, "c_rating_b", rescale_ampere)
apply_func(branch, "c_rating_c", rescale_ampere)
```
(eventually `rescale` should be `rescale_power`)

Reply:
Because the `kv_base` is generally not used by Matpower, I am worried using this field for rescaling will be confusing.  But I can see the utility in the distribution case, lets add an issue and cross this bridge later as the distribution data format is solidified.

Reply:
Done.

In the long run I can imagine the PowerModels data format having more bases, which are used appropriately in cases like this one, but that is out of scope for now.

Reply:
great!